modules/cnf-provisioning-real-time-and-low-latency-workloads.adoc

```diff
@@ -10,6 +10,11 @@ Many industries and organizations need extremely high performance computing and
 
 The cluster administrator can use this performance profile configuration to make these changes in a more reliable way. The administrator can specify whether to update the kernel to kernel-rt (real-time), reserve CPUs for cluster and operating system housekeeping duties, including pod infra containers, and isolate CPUs for application containers to run the workloads.
 
+[WARNING]
+====
+The usage of execution probes in conjunction with applications that require guaranteed CPUs can cause latency spikes. It is recommended to use other probes such as a properly configured set of network probes as an alternative. 
```

Comment:
Maybe offset your example by commas? e.g: 

The usage of execution probes in conjunction with applications that require guaranteed CPUs can cause latency spikes. It is recommended to use other probes, such as a properly configured set of network probes, as an alternative. 


It might help with readability.

Reply:
Good suggestion